website/docs/cloud-docs/api-docs/oauth-clients.mdx

```diff
@@ -235,6 +251,7 @@ Properties without a default value are required.
 | `data.attributes.rsa-public-key`     | string |         | **Required for Bitbucket Data Center in conjunction with the `secret`. Not used for any other providers.** The text of the SSH public key associated with your Bitbucket Data Center application link.   |
 | `data.attributes.organization-scoped` | boolean |         | Whether or not the OAuth client is scoped to all projects and workspaces in the organization. Defaults to `true`.
 | `data.relationships.projects.data[]` | array\[object] | `[]`      | A list of resource identifier objects that defines which projects are associated with the OAuth client. If `data.attributes.organization-scoped` is set to `false`, the OAuth client is only available to this list of projects. Each object in this list must contain a project `id` and use the `"projects"` type. For example, `{ "id": "prj-AwfuCJTkdai4xj9w", "type": "projects" }`. |
+| `data.relationships.agent-pool.data` | object | `{}`    | **For Private VCS** The Agent Pool associated to the VCS connection. This pool will be responsible for forwarding VCS Provider API calls and cloning VCS repositories.                         |
```

Comment:
do we have any other pages that we can link to that explain what private vcs is?

Reply:
Do we _need_ to specify if an attribute is only for Private VCS?

Reply:
> do we have any other pages that we can link to that explain what private vcs is?

AFAIK the only public doc that mentions private-vcs is the [feature entitlements one](https://developer.hashicorp.com/terraform/enterprise/api-docs).

Reply:
Thanks everyone. I went ahead and removed the reference to Private VCS. We can reference this setting in the Private VCS docs instead and keep this page agnostic.